README.rst

```diff
@@ -1,54 +1,67 @@
 FreeType (high-level Python API)
 ================================
 
-Freetype Python provides bindings for the FreeType library. Only the high-level API is bound.
+Freetype Python provides bindings for the FreeType library. Only the
+high-level API is bound.
 
 Documentation available at: http://freetype-py.readthedocs.org/en/latest/
 
 Installation
 ============
 
-There are three ways to go about this.
+**From PyPI, recommended**: `pip install freetype-py`. This will install the
+library with a bundled FreeType binary, so you're ready to go on Windows,
+macOS and Linux (all with 32 and 64 bit support).
 
-1. Recommended: `pip install freetype-py`. This will install the library with a bundled FreeType binary, so you're ready to go on Windows, macOS and Linux (all with 32 and 64 bit support).
-2. If you don't want to or can't use the pre-built binaries, build FreeType yourself: `export FREETYPEPY_BUNDLE_FT=yesplease && python setup.py install`. This will download and compile FreeType with Harfbuzz support as specified in `setup-build-freetype.py`. Set the environment variable `PYTHON_ARCH` to 32 or 64 to explicitly set an architecture, default is whatever your host machine uses. On macOS, we will always build a 96 bit binary.
-    a. Windows: You need CMake and a C and C++ compiler, e.g. the Visual Code Community 2017 distribution with the desktop C++ workload.
-    b. macOS: You need CMake and the XCode tools (full IDE not necessary)
-    c. Linux: You need CMake, gcc and g++. For building a 32 bit library on a 64 bit machine, you need gcc-multilib and g++-multilib (Debian) or glibc-devel.i686 and libstdc++-devel.i686 (Fedora).
-3. Install just the pure Python library and let it find a system-wide installed FreeType at runtime. This is the default.
+Installation with compiling FreeType from source
+------------------------------------------------
 
-Mac users (third way)
----------------------
+If you don't want to or can't use the pre-built binaries, build FreeType
+yourself: `export FREETYPEPY_BUNDLE_FT=yesplease && python setup.py install`.
+This will download and compile FreeType with Harfbuzz support as specified in
+`setup-build-freetype.py`. Set the environment variable `PYTHON_ARCH` to 32 or
+64 to explicitly set an architecture, default is whatever your host machine
+uses. On macOS, we will always build a 96 bit binary.
```

Comment:
Never heard “96 bit”. You mean the “fat” 32/64 bit binary?

Reply:
Yes 😁

Reply:
I think it’s better to avoid that in the Readme.